jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -1,287 +1,70 @@
 
-* ActiveMQ JMS Client Library:
+== Common Use Cases for the Connector
 
-The `activemq-client` of your choice that provides a valid `org.apache.activemq.ActiveMQConnectionFactory` implementation.
+JMS Connector enables you to:
 
-WARNING: This dependency is always required.
+* Listen for new messages as they arrive to the destination.
+* Publish a new message and send it to the specified destination.
+* Consume a message at any given time in the flow from any given destination.
+* Publish a message to any destination, and then wait for a reply on a different destination.
 
-For example, you can use:
+For examples, see xref:jms-examples.adoc[JMS Connector Examples].
 
-[source,xml,linenums]
-----
-<dependency>
-     <groupId>org.apache.activemq</groupId>
-     <artifactId>activemq-client</artifactId>
-     <version>5.14.4</version>
- </dependency>
-----
+== JMS Messaging Support Models
 
-* ActiveMQ Broker Library:
+JMS supports two models for messaging:
 
-The `activemq-broker` is only required when using an in-memory broker based on the VM transport (which is the one configured by default). This dependency should provide a valid `org.apache.activemq.broker.Broker` implementation.
+* Point-to-point queues
+* Publish and subscribe topics
 
-For example, you can use:
+=== Point-to-Point Queues
 
-[source,xml,linenums]
-----
-<dependency>
-    <groupId>org.apache.activemq</groupId>
-    <artifactId>activemq-broker</artifactId>
-    <version>5.14.4</version>
-</dependency>
-----
+In the point-to-point or queuing model, a sender posts messages to a particular queue and a receiver reads messages from the queue. In this model, the sender knows the destination of the message and posts the message directly to the receiver’s queue.
 
-* ActiveMQ KahaDB Library:
+The point-to-point queuing model is as follows:
 
-The `activemq-kahadb-store` is only required when using an persistent in-memory broker based on the VM transport (such as, `+vm://localhost?broker.persistent=true+`).  This dependency provides a valid `org.apache.activemq.store.kahadb.KahaDBPersistenceAdapter` implementation.
+* Only one consumer receives the message.
+* The sender does not need to be running at the time the consumer consumes the message, and the consumer does not need to be running at the time the message is sent.
+* Every message that is successfully processed is acknowledged by the consumer.
 
-For example, you can use:
+=== Publish and Subscribe Topics
 
-[source,xml,linenums]
-----
-<dependency>
-    <groupId>org.apache.activemq</groupId>
-    <artifactId>activemq-kahadb-store</artifactId>
-    <version>5.14.4</version>
-</dependency>
-----
+The publish and subscribe model supports publishing messages to a particular message topic. Subscribers can register to receive messages on a particular message topic. In this model, neither the publisher nor the subscriber are aware of each other. An anonymous bulletin board is a good analogy for this concept.
```

Comment:
```suggestion
The publish and subscribe model supports a sender in publishing messages to a particular message topic rather than directly to a particular queue. Subscribers register to receive messages about a particular message topic, regardless of who sends it. In this model, neither the sender nor the receiver are aware of each other. An anonymous bulletin board is a good analogy for this concept.
```